src/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import StyledAvatar from './styles';
+
+const Avatar = (props) => {
+  const { alt, src, ...attributes } = props;
+
+  if (src) {
+    return (
+      <StyledAvatar {...attributes} >
+        <StyledAvatar.Image alt={alt} src={src} />
+      </StyledAvatar>
+    );
+  }
```

Comment:
I think this condition means that this component is too smart. What would be the case where this Avatar will be rendered without image?

Reply:
Take a look at this examples of composition:
Using the avatar with attributes
```
<Avatar styles>
```
Using with children props
```
<Avatar styles>
   <Image src="/foo/bar">
<Avatar>
```
Here you have a new component `Image` and if you look at styles you already have this two components. May be you can ask. If I need an Avatar with imagem already inside? For this you can create other component and you will have a composition. e.g.
```
import {Avatar, Img} from @memed-dev/epiderme;
 
const AvatarImage = (props) => {
  return (
    <Avatar styles>
      <Image src="{props.src}">
    </Avatar>
  );
}
```